libs/application/templates/document-provider-onboarding/src/forms/PendingReview.ts

```diff
@@ -1,17 +1,72 @@
-import { buildCustomField, buildForm } from '@island.is/application/core'
+import {
+  buildCustomField,
+  buildDescriptionField,
+  buildForm,
+  buildMultiField,
+} from '@island.is/application/core'
 import { ApplicationTypes, Form, FormModes } from '@island.is/application/types'
 
-import { m } from './messages'
+import { m } from '../lib/messages'
 
 export const PendingReview: Form = buildForm({
   id: ApplicationTypes.DOCUMENT_PROVIDER_ONBOARDING,
-  title: m.pendingReviewthankYouScreenTitle,
+  title: '',
```

Comment:
Why remove the form title?

Reply:
This appears on top of the formStepper but since this form only has one step and no stepper it looked weird.